src/content/docs/alerts-applied-intelligence/overview.mdx

```diff
@@ -24,79 +24,94 @@ redirects:
 
-Want to try Alerts & AI? Join us! It's free... forever!
+Alerts allow you to define the conditions in your system you want to be notified about, letting you focus on the things you care about most. Applied intelligence is a machine-learning engine that alerts your team to potential problems by first identifying anomalous behavior, correlating any relevant incidents, alerting your team of active issues and providing a root cause analysis.
 
-<ButtonGroup>
-  <ButtonLink
-    role="button"
-    to="https://newrelic.com/signup"
-    variant="primary"
-  >
-    Sign up for free
-  </ButtonLink>
-</ButtonGroup>
+Alerts and Applied intelligence were created for busy DevOps teams: it reduces noise and makes sure you never miss an issue. 
 
-Already have an account? [Login](https://one.newrelic.com).
+## How it all works 
 
-Alerts lets you define the conditions in your system you want to be notified about, letting you focus on the things you care about most.
+### Detect and diagnose potential problems in your system [#detect-intro]
 
-Applied intelligence is a machine-learning engine that reduces alert noise, correlates incidents, automatically detects anomalies, and provides root cause analysis. Not only that, it enriches your notifications with metadata to give you greater context of what's happening when you are notified.
+New Relic never stops observing your telemetry data and we created alerts so your team can decide when you want to be notified and how.  Applied intelligence takes alerting one step further and monitors for any deviations from what's expected and notifies your team instantly. You can decide what level of automation works best for each situation. Our automatic anomaly detection is easy to use and implement, while our more customizable detection options offer your team the ability to create alerts that are just right for you. 
+
+Learn how to configure anomalies [in our anomaly docs](https://docs.newrelic.com/docs/alerts-applied-intelligence/applied-intelligence/incident-workflows/incident-workflows).
+
+### Correlate relevant issues to reduce noise [#correlation-intro]
+
+After you've detected any irregular behavior in your system, applied intelligence's correlation logic reviews all incoming events and compares them based on signals like time, context, or source. If multiple issues are discovered to be related, instead of creating many, redundant alerts, applied intelligence's correlation logic will group these issues together to create a single, relevant alert. This saves your team time and energy and allows you to get to the heart of the problem faster. 
+
+Our correlation logic is called decisions. Learn how to set up decisions, follow our step by step guide by [in our decisions docs](https://docs.newrelic.com/docs/alerts-applied-intelligence/applied-intelligence/incident-workflows/incident-workflows).
```

Comment:
```suggestion
Our correlation logic is called decisions. Learn how to set up decisions, follow our step by step guide by [in our decisions docs](/docs/alerts-applied-intelligence/applied-intelligence/incident-workflows/incident-workflows).
```

Reply:
Wrong docs link again. Maybe this one works better? https://docs.newrelic.com/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions/